player/src/main/java/com/tpstream/player/util/TrustFactory.kt

```diff
@@ -0,0 +1,47 @@
+package com.tpstream.player.util
+
+import android.content.Context
+import com.tpstream.player.R
+import java.security.KeyStore
+import java.security.cert.Certificate
+import java.security.cert.CertificateFactory
+import javax.net.ssl.SSLContext
+import javax.net.ssl.SSLSocketFactory
+import javax.net.ssl.TrustManagerFactory
+import javax.net.ssl.X509TrustManager
+
+class TrustFactory {
```

Comment:
Add a comment saying let's encrypt dropped support for older android devices with this link https://letsencrypt.org/2023/07/10/cross-sign-expiration